docs/docs/coverage/kubernetes.md

```diff
@@ -0,0 +1,21 @@
+# Kubernetes
+
+When scanning a Kubernetes cluster, Trivy differentiates between the following:
+
+1. Cluster infrastructure (e.g api-server, kubelet, addons)
+1. Cluster configuration (e.g Roles, ClusterRoles). 
+1. Application workloads (e.g nginx, postgresql).
+
+When scanning any of the above, Trivy differentiates between the Kubernetes Resource definition (i.e the YAML that defines it), and the container image if relevant.
+
+Container image is scanned for:
+- Vulnerabilities
+- Misconfigurations
+- Exposed secrets
+
+Kubernetes resource definition is scanned for:
```

Comment:
if mention : `Kubernetes resource is scanned for` maybe should mention `rbac assessment` as well


```suggestion
Kubernetes workload is scanned for:
```

Reply:
this suggestion isn't aligned with the above:

> When scanning any of the above, Trivy differentiates between the Kubernetes Resource definition (i.e the YAML that defines it), and the container image if relevant.

Reply:
Separately, can we write it as "Kubernetes Resource Definition" -- my OCD is very specific hahah